parent/pom.xml

```diff
@@ -1268,6 +1274,17 @@
       </pluginManagement>
       <plugins>
          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <extensions>true</extensions>
+            <configuration>
+               <!-- See configuration details at http://books.sonatype.com/nexus-book/reference/staging-deployment.html -->
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
It should do both.

On Thu, Aug 18, 2016 at 12:24 PM, Sanne Grinovero <notifications@github.com>
wrote:

> In parent/pom.xml
> <https://github.com/infinispan/infinispan/pull/4507#discussion_r75283731>:
>
> > @@ -1268,6 +1274,17 @@
> >        </pluginManagement>
> >        <plugins>
> >           <plugin>
> > +            <groupId>org.sonatype.plugins</groupId>
> > +            <artifactId>nexus-staging-maven-plugin</artifactId>
> > +            <extensions>true</extensions>
> > +            <configuration>
> > +               <!-- See configuration details at http://books.sonatype.com/nexus-book/reference/staging-deployment.html -->
> > +               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
> > +               <serverId>${jboss.releases.repo.id}</serverId>
> > +               <autoReleaseAfterClose>true</autoReleaseAfterClose>
>
> with this setting, will it do both of:
>
>    - close the repository
>    - release it Or only one of these steps?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/infinispan/infinispan/pull/4507/files/3622d5d5f084d439acac5d22cc6a1937862282f7#r75283731>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIBvhvqvd7fUssvLNNYdjClo1EtwGVoCks5qhDLVgaJpZM4Ji91R>
> .
>